Assets/Scripts/GfxReplayPlayer.cs

```diff
@@ -97,10 +97,13 @@ public void ProcessKeyframe(KeyframeData keyframe)
 
                 GameObject instance = Instantiate(prefab);
 
+                // HACK: Gfx-replay scale is currently invalid. We're ignoring it until the issue is resolved.
+                /*
```

Comment:
I have fixed this on the recording side (in habitat-sim), so no need for this on the player side.